src/cmd-buildprep

```diff
@@ -40,6 +40,10 @@ def main():
     builds = None
     if fetcher.exists('builds.json'):
         fetcher.fetch_json('builds.json')
+        # Record the origin and original state
+        with open(BUILDFILES['sourceurl'], 'w') as f:
+            f.write(args.url + '\n')
+        subprocess.check_call(['cp', '-pf', '--reflink=auto', BUILDFILES['list'], BUILDFILES['sourcedata']])
```

Comment:
`shutil.copy2` ? (Though that probably doesn't do `copy_file_range()`, but meh...)

Reply:
Yeah, it doesn't.  I like to use reflinks wherever possible even for "trivial" files so that people remember to use them for bigger stuff too.